src/web/components/Columns/Columns.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Columns from './';
+import Column from '../Column';
+
+describe('Columns', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    const content = (
```

Comment:
what is the purpose of this variable ? Wouldn't it be possible to directly pass the component to `shallow` ?

Reply:
``` 
    wrapper = shallow(<Columns>
      <Column>
        <p>Hello</p>
      </Column>
      <Column>
        <p>world</p>
      </Column>
                      </Columns>);
```

When I do that eslint + prettier  do nonsense. Eslint ask me to respect 4 space of indent and expect the closing tag to be on same the level that the opening tag.

Reply:
Eslint is a different problem, try by running `yarn format` after this refacto

Reply:
With format on save.
![image](https://user-images.githubusercontent.com/8822421/36981337-361f0202-208d-11e8-9617-9215a880c11b.png)
erreur Eslint : Expected indentation of 4 space characters but found 22. (react/jsx-indent)


With yarn run format
> yarn run v1.5.1
$ prettier-eslint --write "pages/**/*.js" && prettier-eslint --write "src/**/*.js"
1 file was unchanged
12 files were unchanged

The result is the same. In addition, when I try to manually modify the file, I always have an eslint error and when I save it, it format the file same as above.

![image](https://user-images.githubusercontent.com/8822421/36981593-ddd6186e-208d-11e8-96e1-495a8d546ba5.png)

Erreur eslint : Expected closing tag to match indentation of opening. (react/jsx-closing-tag-location)

![image](https://user-images.githubusercontent.com/8822421/36981870-945c2a42-208e-11e8-957d-79365ba6cef0.png)
Erreur eslint : Unexpected newline after '('. (function-paren-newline)

That's why i made the content in a const